### ISSUE TITLE
Payment failure

### ISSUE THREAD

**Issue by grandhelmsman** [filecoin-project/go-filecoin#2827]
**Describe the bug**
After the storage requirement file is submitted successfully, the amount is deducted successfully, but the storage miner has not received the amount paid

**To Reproduce**

 go-filecoin client propose-storage-deal t25n5cxbphqoukk43fg5t3fqd6rrobxs7pnlht2zi QmUcTRuyR8Tk1CisskWZrsCdxdmY4KPejp4HodaX82hBY4  0  1000

State:   accepted
Message: 
DealID:  zDPWYqFCvxd7Z9QoGMZgxJVRcDbabAtW12ZHFWaEf1RjtFtJ2PqY

 go-filecoin wallet balance  t1xnehcdnb5kwt33rotzgdsoe5qm5ty6itf7kzo5q
1299.8599999999999994
root@xzo-VirtualBox:~#  go-filecoin client query-storage-deal zDPWYqFCvxd7Z9QoGMZgxJVRcDbabAtW12ZHFWaEf1RjtFtJ2PqY
Status: staged

 go-filecoin client query-storage-deal zDPWYqFCvxd7Z9QoGMZgxJVRcDbabAtW12ZHFWaEf1RjtFtJ2PqY
Status: posted
Message: 

**miner server**
The amount paid by the depositor has not been received，

[root@localhost ~]# go-filecoin config mining.minerAddress
"t25n5cxbphqoukk43fg5t3fqd6rrobxs7pnlht2zi"
[root@localhost ~]#  go-filecoin wallet balance t1usjvldn6hlbhoeph6fzdhojioykzuh6rio3bxga
1398.8995175999999992

**Comment by travisperson** [comment]
I don't believe the payment is paid out to the miner owner at the moment. I could be wrong, but I remember there was something about this brought up a bit ago. 


@ZenGround0 or @acruikshank may have some more information.

**Comment by anorth** [comment]
@acruikshank is the miner expected to claim payment from the channel? If that's not implemented yet, we should have a story for it as part of our drive to storage protocol completion.

**Comment by acruikshank** [comment]
@grandhelmsman @anorth It is __not__ expected that a miner would receive any payment at the time a sector is posted. Currently all the funds for a storage deal are locked up in a payment channel created by the client. The miner will not be able to collect any payment until the block time at which the first voucher (included in the storage deal) becomes valid. At that time the miner will be able to redeem the voucher to collect an amount proportional to the fraction of the storage deal that has elapsed. The miner will not be able to collect full payment until the storage deal is complete.

We cannot automatically redeem payments on behalf of the miner, because it's the miner's choice whether they would rather spend gas to collect sooner or spend less to collect once the deal is done.

We might want to add functionality to ensure the miner automatically collects at the end of the deal, but I wouldn't consider this a priority.

We should have some documentation in the wiki on how a miner can collect payment. It's currently very difficult to collect payment, so this is probably blocked on some UI improvements that will land next release.

**Comment by anorth** [comment]
Right, that makes sense, thank you.

I suggest that we should have a CLI incantation that can collect the
highest voucher for a specific deal (which will probably depend on being
able to list those deals). Do we have existing issues capturing that work,
or should we re-purpose this one into that deliverable?

**Comment by acruikshank** [comment]
The new issue is #2850

**Comment by frrist** [comment]
Closing in favor of #2850.
